lib/chef/knife/kvm_vm_create.rb

```diff
@@ -290,6 +307,9 @@ def bootstrap_node(guest_ip)
 
         wait_for_ssh(guest_ip)
 
+        puts "Bootstrap Config:"
+        puts bootstrap.to_s
+
         begin
```

Comment:
Are the `puts`'s intentional?  It looks like other log messages are `ui.warn` or some such.